src/app/components/profile/pages/profile/my-profile.component.ts

```diff
@@ -94,6 +127,22 @@ class MyProfileComponent
     this.updateStatistics(this.user);
   }
 
+  public toggleAuthTokenVisibility(): void {
+    this.isShowingAuthToken = !this.isShowingAuthToken;
+  }
+
+  public get authToken(): string {
+    return this.sitesApi.getLocalUser()?.authToken;
```

Comment:
why are we asking the sitesApi for a user? Perhaps better to actually inject the source of truth for this?